django/thunderstore/repository/migrations/0008_add_uploader_id.py

```diff
@@ -77,7 +77,7 @@ class Migration(migrations.Migration):
             model_name="packageversion",
             name="file",
             field=models.FileField(
-                storage=backblaze_b2.storage.BackblazeB2Storage(),
+                storage=BACKBLAZE_STORAGE,
```

Comment:
We don't need to (and shouldn't) use any backblaze specific storage here, just use the stub storage like in the `0033_add_s3_cache` migration. The goal is to _standardize_ which storage class is used in migrations (StubStorage in this case) regardless of what the actual environment configuration is during runtime.